src/item_box.py

```diff
@@ -81,6 +82,7 @@ def on_color_dialog_accept(self, dialog, result):
                 self.item.alpha = color.alpha
                 self.update_color()
                 self.application.Clipboard.add()
+                self.application.ViewClipboard.add()
```

Comment:
Changing the color of an item doesn't change the view, so this might not be needed